packages/backend-storage/src/access_builder.test.ts

```diff
@@ -89,19 +93,17 @@ void describe('storageAccessBuilder', () => {
       'unauthenticatedUserIamRole'
     );
   });
-  void it('builds storage access definition for owner', () => {
-    const accessDefinition = roleAccessBuilder.owner.to([
-      'read',
-      'write',
-      'delete',
-    ]);
+  void it('builds storage access definition for IdP identity', () => {
+    const accessDefinition = roleAccessBuilder
+      .id('identity')
```

Comment:
This `.id('identity')` looks confusing and perhaps redundant? From the description

>Currently "identity" is the only option which maps to the Cognito IdP IdentityId. This name was chosen instead of "owner" because it is not actually the user sub from the Cognito UserPool which is the actual "owner" in the system.

I thought `cognito-identity.amazonaws.com:sub` was the userSub. If it's not, can you even access this usersub through [IAM vars](https://docs.aws.amazon.com/cognito/latest/developerguide/amazon-cognito-user-pools-using-the-id-token.html)? All the examples I saw to identify a "user" points to this var.

My recommendation would be to continue to use `owner` which is much more straightforward (and perhaps consistent with what data access rules would look like?) and wait for more "id types" that we might need in the future.

Reply:
This wording was primarily a product decision and the name maps to how the frontend API is going to look. `cognito-identity.amazonaws.com:sub` is not the user sub. It is an identifier that comes from the Identity Pool, not the User Pool.

Reply:
After some offline discussion, we're going to rename this to
```
{
  '/foo/{entity_id}/*': [
    allow.entity('identity').to(['read'])
  ]
}
```